src/content/doc-sdk-golang/methods/create.mdx

```diff
@@ -49,14 +49,116 @@ db.Create[T](table, data)
 ### Example usage
 
 ```go
-// Create an entry
-	person1, err := surrealdb.Create[Person](db, models.Table("persons"), map[interface{}]interface{}{
-		"Name":     "John",
-		"Surname":  "Doe",
-		"Location": models.NewGeometryPoint(-0.11, 22.00),
-	})
+package main
+
+import (
+	"fmt"
+	"time"
+
+	surrealdb "github.com/surrealdb/surrealdb.go"
+	"github.com/surrealdb/surrealdb.go/pkg/models"
+)
+
+//nolint:funlen
+func ExampleCreate() {
+	db := newSurrealDBWSConnection("query", "persons")
```

Comment:
This `newSurrealDBWSConnection(ns, db)` is a test-only helper function that the reader cannot use.
Do you have any ideas on how we can make it more directly translatable to actual application code, without those test helpers? 🤔